roles/ansible/tower/manage-credential-types/README.md

```diff
@@ -12,6 +12,10 @@ A running Ansible Tower with admin permission level access.
 
 The variables used to maintain the Credential Types must be defined in the Ansible Inventory using the `ansible_tower.credential_types` dictionary as explained below.
 
+| Variable | Description | Required | Defaults |
+|:---------|:------------|:---------|:---------|
+|**default_ansible_tower_url**| Ansible Tower url | yes | |
```

Comment:
I'd recommend that we point to the README at the `roles/ansible/tower` level for these as the `default` values are truly just that - i.e.: default values - and the values should be supplied as described at the above mentioned README.